src/helpers/parseAmount.ts

```diff
@@ -2,8 +2,13 @@ import BigNumber from 'bignumber.js';
 import { ELLIPSIS } from 'appConstants';
 import { DECIMALS } from 'config';
 
-export function parseAmount(amount: BigNumber.Value, numDecimals?: number) {
-  const amountAsBigInteger = new BigNumber(amount);
+export function parseAmount(
```

Comment:
can't be use `parseAmount` helper function from sdk-dapp ?

Reply:
uses the same logic as `mx-sdk-js`, but it's less restrictive on the typing side in order to avoid transforms to string in components
will probably change once the deprecated TokenPayment logic in parseAmount is updated in sdk-dapp